src/main/java/de/muenchen/isi/infrastructure/entity/stammdaten/SobonOrientierungswert.java

```diff
@@ -46,9 +21,8 @@ public class SobonOrientierungswertSozialeInfrastruktur extends BaseEntity {
     @Column(nullable = false)
     private Altersklasse altersklasse;
 
-    @Enumerated(EnumType.STRING)
     @Column(nullable = false)
-    private Wohnungstyp wohnungstyp;
+    private String foerderArt;
 
     @Column(precision = 10, scale = 4, nullable = false)
     private BigDecimal einwohnerJahr1NachErsterstellung;
```

Comment:
Die Attribute `faktor1` bzw. `faktor` werden in der Excelvorlage zur Ermittlung des 75-Percentils verwendet.
Die Berechnung des 75-Percentils mit dem Faktor entspricht nämlich nicht dem eigentlichen 75-Percentil (bzw 3.-Quantil).

Die Frage ist, für welche fachlichen Berechnungen der Faktor noch verwendet wird.